tyr/policies/ec2.py

```diff
@@ -18,3 +18,39 @@
         }
      ]
 }"""
+
+allow_mongo_backup_snapshot = """{
```

Comment:
This might be nitpicking but these are more general policies than just for mongo.  maybe a more broad name would be appropriate in case someone wanted to use the same privileges in another role.